app/services/stack_overflow.rb

```diff
@@ -9,30 +9,37 @@ def initialize(term="ruby")
 
   def scrape
     feed = pull_feed
-    if feed.entries.present?
-      format_entries(feed.entries)
+    if feed.present?
+      format_feed(feed)
     end
   end
 
-  def format_entries(entries)
-    entries.map do |entry|
-      formatted_entry = self.format_entry(entry)
+  def format_feed(feed)
+    feed.css('item').map do |item|
```

Comment:
I'm unsure why you changed the name of this method, especially because this method is pretty consistently called format_entries across all services.